drivers/sensor/tdk/icm42688/icm42688.h

```diff
@@ -412,10 +412,10 @@ int icm42688_read_all(const struct device *dev, uint8_t data[14]);
  * @param cfg icm42688_cfg current device configuration
  * @param in raw data value in int32_t format
  * @param out_g whole G's output in int32_t
- * @param out_ug micro (1/1000000) of a G output as uint32_t
+ * @param out_ug micro (1/1000000) of a G output as int32_t
  */
 static inline void icm42688_accel_g(struct icm42688_cfg *cfg, int32_t in, int32_t *out_g,
-				    uint32_t *out_ug)
+				    int32_t *out_ug)
```

Comment:
This function doesn't appear to be used anywhere, @teburd added it. I'd like to wait for his take but I would just remove it since this isn't a public header anyway and there are no Zephyr tree uses of this function.

Reply:
Ack